rust_src/remacs-sys/lib.rs

```diff
@@ -948,6 +948,7 @@ extern "C" {
     pub fn fget_minibuffer_window(f: *const Lisp_Frame) -> Lisp_Object;
     pub fn fget_root_window(f: *const Lisp_Frame) -> Lisp_Object;
     pub fn fget_terminal(f: *const Lisp_Frame) -> *const terminal;
+    pub fn fget_output_method(f: *const Lisp_Frame) -> c_int;
```

Comment:
Add as well:

    pub type output_method = u32;
    pub const output_initial: output_method = 0;
    pub const output_termcap: output_method = 1;
    pub const output_x_window: output_method = 2;
    pub const output_msdos_raw: output_method = 3;
    pub const output_w32: output_method = 4;
    pub const output_ns: output_method = 5;

This way you do not have magic numbers in `framep` and you avoid the C enum issue @birkenfeld raised.

Reply:
yes it is ugly. and annoying to hand type out those numbers. I'd love a macro for it.

Reply:
One more suggestion would be to define the enum nevertheless, and use one of the crates that generate a `TryFrom<int>` impl for it automatically.

Reply:
Sounds more elegant, but is this worth the effort considering that we want to remove the c structs eventually ? It seems to me, that we will have many "ugly solutions" while there's c code. 

And here the constant solution looks simple and explicit.

Reply:
No real preference, just an idea I had :)